dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/CommandLine.java

```diff
@@ -81,6 +82,19 @@ static ArgumentParser getArgumentParser() {
                 .type(Boolean.class)
                 .setDefault(false)
                 .help("Only update image tag store. Skip creating PRs");
+        parser.addArgument("-u", "--" + Constants.USE_RATE_LIMITING)
+                .type(Boolean.class)
+                .setDefault(false)
+                .help("Use RateLimiting when sending PRs");
+        parser.addArgument("-r", "--" + Constants.RATE_LIMIT)
+                .type(Long.class)
+                .help("Max number of PRs to be sent per " + Constants.RATE_LIMIT_DURATION);
+        parser.addArgument("-rl", "--" + Constants.RATE_LIMIT_DURATION)
+                .type(Duration.class)
+                .help("Rate limit duration");
```

Comment:
Let's consolidate this into the rate limit flag. You can represent durations via a set of characters such as `100m` (100 per minute), `500s` (500 per second), etc.

Reply:
Can you please point me to some example for this? As far as I understand, duration can represent the time part only and not the rate https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence- so 100m would mean 100minutes, etc. 
Also, the Refill methods where this will be used, expects the duration and rate as separate arguments https://javadoc.io/doc/com.github.vladimir-bukhtoyarov/bucket4j-core/4.10.0/io/github/bucket4j/Refill.html . If we try to club them during input, then we would have to parse them as well at later stage.

Reply:
We shouldn't pass through Refill's interface and provide a simpler, more constrained one